src/main/java/com/exadel/ipromise/dao/impl/UserDaoImpl.java

```diff
@@ -0,0 +1,65 @@
+package com.exadel.ipromise.dao.impl;
+
+import com.exadel.ipromise.dao.UserDao;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import com.exadel.ipromise.entity.User;
+
+import java.sql.PreparedStatement;
+
+@Repository
+public class UserDaoImpl implements UserDao {
```

Comment:
Using interface looks good! We must use interfaces everywhere - it's good practice

Reply:
Done